modules/querying/pages/select-statement/index.adoc

```diff
@@ -437,7 +437,7 @@ For example, consider the following clauses:
 
 [source,gsql]
 ----
-FROM Person:A -(Knows.Knows)- Person:C
+FROM (A:Person) -[Knows.Knows:Knows]- (C:Person)
```

Comment:
```suggestion
FROM (A:Person) -[:Knows*2]- (C:Person)
```
In GSQL V2,  `-(Knows.Knows)-` is a shorthand to say "a `Knows` edge followed by another `Knows` edge, and no restriction on the vertex in between them, e.g.
(Person A) -[Knows]-> (_anything_) -[Knows]-> (Person C)

Cypher does not support this *dot* shorthand for `edge1.edge2.edge3.`
Cypher does support a *multiplier* shorthand, as shown above.